rasa_sdk/interfaces.py

```diff
@@ -57,7 +57,10 @@ def __init__(
         self.latest_action_name = latest_action_name
 
     def current_state(self) -> Dict[Text, Any]:
-        """Return the current tracker state as an object."""
+        """Return a dict containing information about the current
```

Comment:
Maybe for this and the utter_message one, since they either have a lot of args or a complicated return object, it makes sense to do the docstrings like formatted at https://github.com/RasaHQ/rasa-sdk/pull/128/files#diff-74589aedc0a7e9886c46b20e8be0c101R165?